bot.py

```diff
@@ -1,6 +1,18 @@
 __version__ = "4.0.2"
 
 
+# early import to initialize colorama
+from core.models import (
+    DMDisabled,
+    HostingMethod,
+    InvalidConfigError,
+    PermissionLevel,
+    SafeFormatter,
+    create_log_handler,
+    configure_logging,
+    getLogger,
+)
```

Comment:
This doesn't look that great, usually you import local packages after stdlib and external packages. Hence, [we initialized colorama within this file, before the local imports](https://github.com/modmail-dev/modmail/pull/3216/files?diff=split&w=0#diff-36f6200cd8353111badb874140eba3af6055f4cc5839fbdeb9ede992620828c1L28-L34).